types/lodash/index.d.ts

```diff
@@ -11620,15 +11620,15 @@ declare namespace _ {
          */
         cloneDeepWith<TResult>(
```

Comment:
`TResult` is now only used in one place. Should probably just be `any`. We have a lint rule for this `no-unnecessary-generics` but it is disabled in lodash right now.